data/human/human missions.txt

```diff
@@ -7453,3 +7453,233 @@ mission "Adelita 5"
+			`		The commercial finance business traditionally is extremely competitive. Our Finance segment is subject to competition from various types of financing institutions, including banks, leasing companies, commercial finance companies and finance operations of equipment vendors. Competition within the commercial finance industry primarily is focused on price, term, structure and service.`
+			`		Megaparsec's diverse portfolio and commitment to innovation position us well for future growth, and we remain dedicated to enhancing value for our shareholders while serving our customers' needs.`
+			`	You've finally gotten through the report's opening statements. That's more than enough of the main report to go through.`
+			choice
+				`	(Turn to Section 3, Part C, Paragraph 6 that Howard highlighted earlier.)`
+					goto sectionmarked
+				`	(Close out the report now that you've gone through the main overview.)`
+					decline
+			label sectionmarked
+			`	You open the attached report to the area indicated by Howard:`
+			branch goodchoicesec2
+				has "Syndicate Business expansion 10m good"
+			branch evilchoicesec2
+				has "Syndicate Business expansion 10m evil"
+			# neutral 2
+			`	Crossroads refueling station investments continue to meet projections, with healthy margins and steady returns. Following several modest investments in operational efficiency, returns are tracking slightly above baseline forecasts, ensuring a consistent annualized return to investors. The station's strategic position on the centrally located planet continues to generate strong traffic, further solidifying its long-term profitability.`
+				goto reportend
+			label goodchoicesec2
+			`	Investment returns from the Crossroads refueling station are currently below comparable benchmarks. However, environmental-related legal actions have been significantly lower than anticipated, resulting in reduced provisions for legal fees. Additionally, localized employee turnover has halved during this period, contributing to enhanced operational stability and providing a favorable outlook for the coming quarters.`
+				goto reportend
+			label evilchoicesec2
+			`	A serious accident at the Crossroads refueling station resulted in injuries to two dozen employees and one fatality. While the legal ramifications are being addressed, the station's outsized returns have more than compensated for the legal fees incurred due to this unfortunate and unpredictable occurrence. Financial projections remain strong, with returns far exceeding expectations.`
+			label reportend
+			`	That marks the end of where the report covers your investment. You wonder what upcoming "investment opportunity" Howard has in mind.`
+				decline
+
+mission "Syndicate Business 5.1 B 10m Expansion Crossroads Update"
+	# landing   not landing because expansion is by spaceport
```

Comment:
Did you forget to take this out before pushing up?

Reply:
Yup

Reply:
```suggestion
```